rust-toolchain

```diff
@@ -1 +1 @@
-1.89.0
+1.92.0
```

Comment:
Should we upgrade the individual packages too, while we are at it? Or that's risky so we do that on a separate PR?